### PR TITLE
initial code for spring-cloud-etcd-config and small fixes

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -141,7 +142,7 @@
             <dependency>
                 <groupId>org.springframework.cloud</groupId>
                 <artifactId>spring-cloud-netflix-core</artifactId>
-                <version>1.0.0.BUILD-SNAPSHOT</version>
+                <version>${spring.cloud.version}</version>
```
Comment (spencergibb) [inline pom.xml]: These don't have the same versions anymore. See [this pom.xml](https://github.com/spring-cloud/spring-cloud-consul/blob/master/pom.xml#L60-L62) in s-c-consul for an example.

**Comment by spencergibb** [comment]
Looking good.

**Comment by spencergibb** [comment]
Have you signed the [contributor agreement](https://support.springsource.com/spring_committer_signup)?

**Comment by lburgazzoli** [comment]
Fixed POMs, who is Spring Cloud's project lead ?

**Comment by spencergibb** [comment]
Myself or Dave Syer (@dsyer)

**Comment by lburgazzoli** [comment]
Contributor agreement signed

**Comment by spencergibb** [comment]
thanks!

**Pull request merged**